Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -214,26 +219,39 @@ class Bar < Formula
     bar_file.unlink unless bar_file.nil?
   end
 
-  def test_doctor_check_path_for_trailing_slashes
-    assert_match "Some directories in your path end in a slash",
-      cmd_fail("doctor", "check_path_for_trailing_slashes",
-               {"PATH" => ENV["PATH"] + File::PATH_SEPARATOR + "/foo/bar/"})
+  def test_doctor
+    # check_for_non_prefixed_coreutils depends on coreutils.rb's existence and
+    # check_for_non_prefixed_findutils on findutils.rb's one
+    repo = CoreFormulaRepository.new
+    coreutils_file = repo.formula_dir/"coreutils.rb"
+    findutils_file = repo.formula_dir/"findutils.rb"
+    coreutils_file.write <<-EOS.undent
+      class Coreutils < Formula
+        url "https://example.com/foo-1.0.tar.gz"
+      end
+    EOS
+    findutils_file.write <<-EOS.undent
+      class Findutils < Formula
+        url "https://example.com/foo-1.0.tar.gz"
+      end
+    EOS
+    assert_match "This is an integration test",
+                 cmd_fail("doctor")
```

Comment:
My concerns are:
* brew doctor itself does take some time. Especially for check Homebrew outdate 
* It's already be run `test-bot`. 
* The result will be dependent to running environment, which is probably not what we want inside test case.